### DISCUSSION TITLE
helm-bibtex doesn't work from everywhere when org-roam-bibtex-mode enabled

### DISCUSSION THREAD

**Discussion by P451M** [org-roam/org-roam-bibtex#267] [Q&A] [answered]
Hi, 

I'm currently getting started with annotated bibliography management with *Spacemacs* and am experimenting with *org-roam-bibtex*. I face the following problem:

1. When `org-roam-bibtex-mode` is disabled I can run `helm-bibtex` within any mode to open the `helm-bibtex` buffer. 
2. When `org-roam-bibtex-mode` is enabled, `helm-bibtex` only seems to work within `org-mode` or `bibtex-mode`. In other modes, as for instance in the *messages* buffer, I get the error: 
`helm-M-x-execute-command: Args out of range: #<buffer *Messages*>, 0, 0` 
 
Is it possible to use `helm-bibtex` from anywhere having `org-roam-bibtex-mode` enabled? Or is that not supported?

**Comment by myshevchuk** [comment, marked as answer]
Hi @P451M , thank you for reporting this. It is indeed a bug caused by ORB. Please open [a new issue](https://github.com/org-roam/org-roam-bibtex/issues/new/choose).

**Reply by P451M** [reply]
Thanks for the reply @myshevchuk! Issue is opened :).
